ludwig/features/audio_feature.py

```diff
@@ -452,15 +453,13 @@ def _get_max_length_feature(preprocessing_parameters, sampling_rate_in_hz, audio
 
 @register_input_feature(AUDIO)
 class AudioInputFeature(AudioFeatureMixin, SequenceInputFeature):
-    encoder = "parallel_cnn"
-    max_sequence_length = None
-    embedding_size = None
+    encoder = {TYPE: "parallel_cnn", "max_sequence_length": None, "embedding_size": None}
```

Comment:
In a follow-up PR, we'll want to change the encoder here from a dict to something using the schema.